trace/cloud-trace-demo-app/README.md

```diff
@@ -0,0 +1,48 @@
+# cloud-trace-demo-app
+
+Open this demo app in [Google Cloud Shell](https://cloud.google.com/shell/docs/). This includes necessary tools.
+
+We provide a public image for the services in this demo app. You could also build
+your own following step 4 - 6.
+
+[![Open Cloud Trace Demo APP in Cloud Shell](http://gstatic.com/cloudssh/images/open-btn.svg)](https://console.cloud.google.com/cloudshell/open?cloudshell_git_repo=https://github.com/GoogleCloudPlatform/python-docs-samples&page=editor&open_in_editor=trace/cloud-trace-demo-app/README.md&amp;cloudshell_tutorial=README.md)
+
+#### Demo Requirements
+If you are using Cloud Shell, skip to the next section.
+
+1. Install gcloud <https://cloud.google.com/sdk/install>
+2. Install kubectl <https://kubernetes.io/docs/tasks/tools/install-kubectl/>
+3. Install docker <https://docs.docker.com/install/>
+
+#### Google Container Registry Image Setup
+If you are using the provided image, skip to the next section.
+
+4. Get default project id and set environment variable:
+    `PROJECT_ID=$(gcloud config get-value project)`
+5. Build Image:
+    `docker build -t gcr.io/${PROJECT-ID}/cloud-trace-demo .`
+6. Upload Image to Container Registry:
+    `gcloud docker -- push gcr.io/${PROJECT-ID}/cloud-trace-demo-test:v1`
+
+#### Create a GKE cluster
+7. Enable Google Cloud and set up region and zone.
+    `gcloud init`
+8. Enable the GKE api & billing:
+    `gcloud services enable container.googleapis.com`
+9. Create a GKE cluster named "demo":
+    `gcloud container clusters create demo`
+
+#### Send Requests to See Generated Traces
+
+10. If you are using your own image, please change the image variable in the following files:
+    * [YAML](./app/demo-service-a.yaml)
+    * [template B](./app/demo-service-b.yaml.template)
+    * [template C](./app/demo-service-c.yaml.template)
+11. Run setup.sh to apply the YAML files. Note this file configures the default image we provide.
+    `./setup.sh`
+12. Send request to the last service:
+    `curl $(kubectl get svc cloud-trace-demo-a -ojsonpath='{.status.loadBalancer.ingress[0].ip}')`
+13. Visit [Trace List](https://pantheon.corp.google.com/traces/list) to check traces generated
+    ![Screenshot](./example.png)
```

Comment:
Yay! I'm glad you have a screenshot - a few nits though
- remove your picture from the top right
- remove the part with your project name since it has your name in it
- add a note about what folks should notice in the image - will their screenshot look identical to yours or similar? What should they be seeing in this page?

Reply:
Done.

Reply:
So close!!! Minor nits:
* rephrase "Click on Trace" to say "Click on any trace in the graph to see the Waterfall View"
* Remove the words from the screenshot, but leave the circle and the arrow